backend/src/main/java/com/festago/student/dto/StudentResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.festago.student.dto;
+
+import com.festago.school.domain.School;
+
+public record StudentResponse(
+    boolean isVerified,
+    StudentSchoolResponse school
+) {
+
+    public static StudentResponse verified(School school) {
+        return new StudentResponse(
+            true,
+            StudentSchoolResponse.from(school)
+        );
+    }
+
+    public static StudentResponse notVerified() {
+        return new StudentResponse(
+            false,
+            null
+        );
+    }
```

Comment:
인증이 되지 않은 사용자는 해당 정적 메서드를 호출하여 응답을 반환하는군요!
하지만 매번 새로운 인스턴스 값을 넘겨줄 필요가 있을 것 같지는 않아요! (record 클래스가 불변하므로)
따라서 다음과 같이 캐싱된 값을 넘겨줘도 좋을 것 같네요!
```suggestion
    private static final StudentResponse NOT_VERIFIED = new StudentResponse(false, null);
    ...
    public static StudentResponse notVerified() {
        return NOT_VERIFIED;
    }
```

Reply:
글렌이 제안해주신 방법이 캐싱뿐만 아니라 상수 형식으로 빼놓기에 전 '가독성 측면에서도 더 좋을 수 있겠다' 라는 생각도 드네요

Reply:
오, 가독성과 캐싱 두가지 이점을 모두 챙길 수 있겠군요! 반영하겠습니다